docs/pages/operations-guide.md

```diff
@@ -10,123 +10,149 @@ redirect_from: "/ops-guide"
-    1. Cosmos creates a Marathon app definition by rendering the package's `marathon.json.mustache` with the configuration options provided in the request. In the case of an SDK service, this app represents the service's Scheduler. Cosmos queries Marathon to create the app.
-    1. Marathon launches the service's Scheduler somewhere in the cluster, using the rendered app definition provided by Cosmos.
-    1. The service Scheduler is launched. From this point onwards, the deployment is being handled by the SDK.
+1. Cosmos creates a Marathon app definition by rendering the package's `marathon.json.mustache` with the configuration options provided in the request. In the case of an SDK service, this app represents the service's Scheduler. Cosmos queries Marathon to create the app.
 
-The service Scheduler's `main()` function is run like any other Java application. At this point the Scheduler has the following state to start from:
-- `svc.yml` template representing the service configuration.
+1. Marathon launches the service's Scheduler somewhere in the cluster using the rendered app definition provided by Cosmos.
+
+1. The service Scheduler is launched. From this point onwards, the SDK handles deployment.
+
+The service Scheduler's `main()` function is run like any other Java application. At this point, the Scheduler has the following state to start from:
+- `svc.yml` template that represent the service configuration.
 - Environment variables provided by Marathon, to be applied onto the `svc.yml` template.
 - Any custom logic implemented by the service developer in their Main function (we'll be assuming this is left with defaults for the purposes of this explanation)
 
-1. The Scheduler uses the above pieces to bootstrap itself into a running service as follows:
-    1. The `svc.yml` template is rendered using the environment variables provided by Marathon.
-    1. The rendered `svc.yml` "Service Spec" contains the host/port for the Zookeeper instance which the Scheduler uses for persistent configuration/state storage. The default is `master.mesos:2181`, but may be manually configured to use a different Zookeeper instance. The Scheduler always stores its information under a ZK node named `dcos-service-<svcname>`.
-    1. The scheduler connects to that Zookeeper instance and checks to see if it's previously stored a Mesos Framework ID for itself.
-        - If the Framework ID is present, the Scheduler will attempt to reconnect to Mesos using that ID. This may result in a "Framework has been removed" error if Mesos doesn't recognize that Framework ID, indicating an incomplete uninstall.
-        - If the Framework ID is not present, the Scheduler will attempt to register with Mesos as a Framework. Assuming this is successful, the resulting Framework ID is then immediately stored.
+#### The Scheduler uses the above pieces to bootstrap itself into a running service
```

Comment:
`Steps handled by the Scheduler`? To mirror the prior section for steps handled by DC/OS